src/main/java/de/tum/in/www1/artemis/domain/TextExercise.java

```diff
@@ -35,17 +34,12 @@ public void setExampleSolution(String exampleSolution) {
         this.exampleSolution = exampleSolution;
     }
 
-    @JsonIgnore
-    public boolean isFeedbackSuggestionsEnabled() {
-        return getAssessmentType() == AssessmentType.SEMI_AUTOMATIC;
-    }
-
     /**
      * Disable feedback suggestions for this exercise by setting the assessment type to MANUAL.
      * Only changes the assessment type if feedback suggestions are currently enabled.
      */
     public void disableFeedbackSuggestions() {
-        if (isFeedbackSuggestionsEnabled()) {
+        if (this.getFeedbackSuggestionsEnabled()) {
             setAssessmentType(AssessmentType.MANUAL);
         }
     }
```

Comment:
Why don't we support feedback suggestions for text exam exercises? I just saw `disableFeedbackSuggestionsForExamExercises` , also what about programming exercises?

Reply:
It wasn't supported before, and I didn't want to introduce it with this PR because there might be some unexpected problems. This PR is already complex enough IMO. I now also disable it for imported programming exercises. The option was already unavailable for both.